### DISCUSSION TITLE
Refreshing the device definitions

### DISCUSSION THREAD

**Discussion by Milne77** [Koenkk/zigbee2mqtt#7968] [General]
Since i installed zigbee2mqtt, new tools have appeared in the list of supported devices (on the site: [https://www.zigbee2mqtt.io/information/supported_devices.html](https://www.zigbee2mqtt.io/information/supported_devices.html)).
How can I get an already installed system to download the new definition list? 
Zigbee2mqtt runs on raspberry, linux, as a plugin for the loxberry framework.

**Comment by Koenkk** [comment]
This can be done by updating z2m: https://www.zigbee2mqtt.io/getting_started/running_zigbee2mqtt.html#6-for-later-update-zigbee2mqtt-to-the-latest-version

**Reply by Milne77** [reply]
Dear Koenk!
It's great, thank you! :)
